extra-hatch-configuration/requirements-dev-packages.txt

```diff
@@ -4,6 +4,8 @@ pre-commit>=3.2.2, <3.3.0
 pytest-mock>=3.7.0, <3.8.0
 pytest-xdist>=3.2.1, <3.3.0
 pytest>=7.1.1, < 7.2.0
+PyYAML>=6.0, <7.0.0
+graphviz>=0.18.2, <0.21
```

Comment:
These aren't dev dependencies, are they? I'm not sure why we need pyYAML, but graphviz is referenced in our production package. As such, it should probably be in the metricflow-semantics dependency set, or at least listed as an optional dependency there.

Reply:
`pyYAML` is needed to parse YAML configuration files for tests. `graphviz` is used in the tests and the CLI. Let me put it back as a regular dependency until I figure out what to do with it.